composer.json

```diff
@@ -6,7 +6,7 @@
     "type": "magento2-theme",
     "require": {
       "magento/framework": "*",
-      "snowdog/module-alpaca-components": "dev-develop",
+      "snowdog/module-alpaca-components": "dev-feature/36717",
```

Comment:
I'm not sure about the flow, but should we change the version here ?

Reply:
I have the changes on this branch, and to see the effect when working in the theme I must change the target here. When it will be merged into test or develop i will change it to the right one again